test/prototype_transforms_kernel_infos.py

```diff
@@ -400,6 +413,23 @@ def _full_affine_params(**partial_params):
 ]
 
 
+def _get_fills(*, num_channels, dtype, vector=True):
```

Comment:
`fill` was creating issues because we set something like `128.0` for floating point images. This is not an issue in general, but affects stuff like aggregated diff. This just generalizes the fill generation for all tests below.